rest/codegen/paths_test.go

```diff
@@ -0,0 +1,143 @@
+package codegen
```

Comment:
There should be tests for the different types as well - in particular slices (Array) once they are supported.